### PR TITLE
Non-CR3 uploads enhancement - validate lat/lons

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes cityofaustin/atd-data-tech/issues/8248

## Testing
**URL to test:** 
https://deploy-preview-1155--atd-vze-staging.netlify.app/#/tools/upload_non_cr3

**Steps to test:**
You can use [this](https://github.com/cityofaustin/atd-vz-data/files/10396790/non_cr3_template.csv) file for testing
1. Go to the Non-CR3 Uploads page under Tools
2. Upload the above .csv
3. The second to last record is a coordinate pair with the lat/longs switched, it should return an error.
4. The last record is a coordinate pair outside of the Austin bounding box, it should return an error.

Please send critique about the error message/if it could be more helpful or concise. I feel like it could be more descriptive but don't want it to be too long or break pattern with the other error messages?

---
#### Ship list
- [x] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

**Review by chiaberry — APPROVED** [review]
I like the extra validation you added and our discussion about it yesterday in planning

File: atd-vze/src/views/Tools/ToolsUploadNonCR3.js [hunk 1]
```diff
@@ -163,7 +163,9 @@ const ToolsUploadNonCR3 = () => {
     }
 
     if (!isValidCoord(record["longitude"], record["latitude"])) {
-      errors.push("Invalid coordinate pair");
+      errors.push(
+        "Invalid coordinate pair (for example, -97.7404, 30.2747 is a valid Austin coordinate pair)"
```
Comment (frankhereford) [inline atd-vze/src/views/Tools/ToolsUploadNonCR3.js:167]: Thanks for adding this. An example is always helpful for me when I'm trying to match the form of something.

**Review by frankhereford — APPROVED** [review]
This looks awesome - thank you Rose!

As an aside, this is the first time I have actually tried to upload a file into this interface, and I really kinda like [that table component](https://handsontable.com/) on there. Going to remember that for next time we have a need for showing a lot of tabular data in a small footprint.

**Review by johnclary — APPROVED** [review]
Works like a charm. I really like this feature—awesome work! ✨ 🚢 

I did notice some bugginess when trying to close the success confirmation modal. I imagine this bug pre-dates this work. [Screen cap](https://user-images.githubusercontent.com/14793120/212133375-b1a36b99-026b-4895-9ca6-c2bf1cd2c628.mov)

Debugging that ☝️ would be a nice opportunity to move some of the code crammed into this component into a config and/or utilities. I totally understand y'all are trying to ship and this is prob not a priority.

File: atd-vze/src/views/Tools/ToolsUploadNonCR3.js (hunk 1 above)
Comment (johnclary) [reply atd-vze/src/views/Tools/ToolsUploadNonCR3.js:167]: 💯

**Comment by roseeichelmann** [comment]
@johnclary lol oh man gotta love the infinite closing bug. maybe we can make a follow up issue for this @patrickm02L ?

**Review by patrickm02L — APPROVED** [review]
The correct example of the coordinates in the error message is a nice addition! Good to go. 🤘

Screen Shot 2023-01-17 at 9 16 11 AM

**Comment by patrickm02L** [comment]
> @johnclary lol oh man gotta love the infinite closing bug. maybe we can make a follow up issue for this @patrickm02L ?

Issue created #11210

**Pull request merged**